src/lib/components/Werkvorm.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+	import ArrowIcon from './icons/ArrowIcon.svelte';
+
+	export let werkvormTitle: string;
+	export let werkvormDescription: string;
+	export let werkvormImage: string = '/assets/images/placeholder.png';
+	export let werkvormLink: string;
```

Comment:
Suggestiel: gebruik korte descriptive props (title, description, image, link) meer leesbaar is meer beter